src/admin-views/tribe-options-tickets.php

```diff
@@ -37,50 +37,88 @@
 	'event-tickets_rsvp' => __( 'Require users to log in before they RSVP', 'event-tickets' ),
 ) );
 
-$tickets_tab = array(
-	'priority' => 20,
-	'fields' => apply_filters(
-		'tribe_tickets_settings_tab_fields',
-		array(
-			'tribe-form-content-start' => array(
-				'type' => 'html',
-				'html' => '<div class="tribe-settings-form-wrap">',
-			),
-			'tickets-title' => array(
-				'type' => 'html',
-				'html' => '<h3>' . esc_html__( 'Ticket Settings', 'event-tickets' ) . '</h3>',
-			),
-			'ticket-enabled-post-types' => array(
-				'type' => 'checkbox_list',
-				'label' => esc_html__( 'Post types that can have tickets', 'event-tickets' ),
-				// only set the default to tribe_events if the ticket-endabled-post-types index has never been saved
-				'default' => array_key_exists( 'ticket-enabled-post-types', $options ) ? false : 'tribe_events',
-				'options' => $all_post_types,
-				'validation_type' => 'options_multi',
-				'can_be_empty' => true,
-			),
-			'ticket-authentication-requirements-heading' => array(
-				'type' => 'html',
-				'html' => '<h3>' . __( 'Login requirements', 'event-tickets' ) . '</h3>',
-			),
-			'ticket-authentication-requirements-advice' => array(
-				'type' => 'html',
-				'html' => '<p>' . sprintf(
-						__( 'You can require that users log into your site before they are able to RSVP (or buy tickets). Please review your WordPress Membership option (via the General Settings admin screen) before adjusting this setting.', 'event-tickets' ),
-						'<a href="' . get_admin_url( null, 'options-general.php' ) . '" target="_blank">',
-						'</a>'
-					) . '</p>',
-			),
-			'ticket-authentication-requirements' => array(
-				'type' => 'checkbox_list',
-				'options' => $ticket_addons,
-				'validation_type' => 'options_multi',
-				'can_be_empty' => true,
-			),
-			'tribe-form-content-end' => array(
-				'type' => 'html',
-				'html' => '</div>',
-			),
-		)
+
+$ticket_form_location_options = array(
+	'tribe_events_single_event_after_the_meta'     => __( 'Below the event details [default]', 'event-tickets' ),
+	'tribe_events_single_event_before_the_meta'    => __( 'Above the event details', 'event-tickets' ),
+	'tribe_events_single_event_after_the_content'  => __( 'Below the event description', 'event-tickets' ),
+	'tribe_events_single_event_before_the_content' => __( 'Above the event description', 'event-tickets' ),
+);
+
+$tickets_fields = array(
+	'tribe-form-content-start'                   => array(
```

Comment:
Minor spacing thing ... `=> array(` is unusually far out here: in cases like this, is it worth trying to align them? If we _do_ want to align them, we might want to make some tweaks for consistency (ie look down to the `ticket-rsvp-form-location` key/line 65).